src/Uno.UI/Helpers/UnoRuntimePlatform.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Uno.UI.Helpers;
+
+internal enum UnoRuntimePlatform
+{
+	Android,
+	iOS,
+	MacCatalyst,
+	MacOSX,
+	WebAssembly,
+	Windows,
+	Skia,
+	SkiaGtk,
+	SkiaWpf,
+	SkiaX11,
+	SkiaFrameBuffer,
+	SkiaMacOS,
```

Comment:
When\where\how would `Skia*` (e.g. `SkiaMacOS`) be returned ?

There's no code to allow to set it (from the host) since it's returned from a method. An internal `Set...` method (only for `__SKIA__` could be used by the different hosts).

Also if `Skia*` values are returned when would the lone `Skia` value be returned ?

You might want to add an extension method, on the enum, like `IsSkia()` to make it easier to detect all Skia-based platforms (and remove `Skia`).

Reply:
This sounds good, in some cases you want to make some code skia specific (maybe because of native rendering limitations), in other cases it is the "native platform" that matters only; so a combination of `IsSkia()` and `GetPlatform()` makes sense; and dropping `Skia` enum value completely (and use unknown as the "default")